cocos/gfx/webgl/webgl-descriptor-set.ts

```diff
@@ -55,12 +55,13 @@ export class WebGLDescriptorSet extends DescriptorSet {
         for (let i = 0; i < bindings.length; ++i) {
             const binding = bindings[i];
             for (let j = 0; j < binding.count; j++) {
-                gpuDescriptors.push({
+                const gpuDescriptor: IWebGLGPUDescriptor = {
                     type: binding.descriptorType,
                     gpuBuffer: null,
                     gpuTexture: null,
                     gpuSampler: null,
-                });
+                };
+                gpuDescriptors.push(gpuDescriptor);
```

Comment:
This change has to be done to make mangle work, since mangling properties needs to know the type of interface.
It's difficult to get the type of `{...}` without define it explicitly because it just a pure `__object` type.